From de4e489cd7d32f92326cce08e626c0e67ab16ee4 Mon Sep 17 00:00:00 2001
From: Christian Gagneraud <chgans@gna.org>
Date: Sat, 28 Apr 2012 21:23:05 +0100
Subject: [PATCH] Fix documentation following patches that fix the missing
 initialisations in the attitude structure

A couple of patches have been send recently that initialiasize the temperature
and water depth to NAN when the information is not available.
This has changed the JSON output for certains devices.
This patch fixes documentation and testsuites accordingly.

Signed-off-by: Christian Gagneraud <chgans@gna.org>
Signed-off-by: Eric S. Raymond <esr@thyrsus.com>
---
 gpsd_json.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpsd_json.xml b/gpsd_json.xml
index 35a9df2..25c6bde 100644
--- a/gpsd_json.xml
+++ b/gpsd_json.xml
@@ -793,7 +793,7 @@ will assert ATT_IS.</para>
 {"class":"ATT","tag":"PTNTHTM","time":1270938096.843,
     "heading":14223.00,"mag_st":"N",
     "pitch":169.00,"pitch_st":"N", "roll":-43.00,"roll_st":"N",
-    "dip":13641.000,"mag_x":2454.000,"temperature":0.000,"depth":0.000}
+    "dip":13641.000,"mag_x":2454.000}
 </programlisting>
 </listitem>
 </varlistentry>
-- 
1.7.10

